### PR TITLE
Oxidation api types bindings

### PR DESCRIPTION
**Pull request by Max-7**
Includes #1924

### CODE REVIEW

File: tests/event_bus_spy.py [hunk 1]
```diff
@@ -169,7 +209,7 @@ def assert_events_occured(self, events, in_order=True):
 
     def assert_events_exactly_occured(self, events):
         events = self._cook_events_params(events)
-        assert self.events == events
+        assert events == self.events
```
Comment (touilleMan) [inline tests/event_bus_spy.py:212]: Is the change related to the new `__eq__` ? If so a comment should be added

File: tests/test_types.py [hunk 2]
```diff
@@ -70,27 +74,26 @@ def test_backend_addr_good(url, expected):
...
             "parsec://foo:42/dummy",
             "Cannot have path",
         ),
         (
-            # bad parsing in unknown param
-            "parsec://foo:42?dummy",
-            "bad query field: 'dummy'",
-        ),
-        (
+            #            # bad parsing in unknown param
```
Comment (touilleMan) [inline tests/test_types.py:90]: Should add a comment or remove the test entirely ?

File: tests/test_types.py [hunk 3]
```diff
@@ -152,19 +158,18 @@ def test_backend_organization_addr_good(base_url, expected, verify_key):
...
             "xx://foo:42/org?rvk=<rvk>",
             "Must start with `parsec://`",
         ),
         (
-            # bad parsing in unknown param
-            "parsec://foo:42/org?rvk=<rvk>&dummy",
-            "bad query field: 'dummy'",
-        ),
-        (
+            #            # bad parsing in unknown param
```
Comment (touilleMan) [inline tests/test_types.py:169]: Same as above

**Review by touilleMan — APPROVED** [review]
It's time for merge&pray :trollface:

File: tests/event_bus_spy.py (hunk 1 above)
Comment (Max-7) [reply tests/event_bus_spy.py:212]: Yes, it's related to the change made in https://github.com/Scille/parsec-cloud/pull/1950/files#diff-65728c3d9bd60f81308060373b1016f2f7cadc9583db97f29f5ebe3b6b7db1c1R60

**Pull request merged**